Next/src/pages/support.tsx

```diff
@@ -0,0 +1,181 @@
+// flex-row
+// each icon (wrapper)
+
+import type { NextPage } from "next";
+import Head from "next/head";
+import Image from 'next/image';
+import styled from "styled-components";
+
+const PageContainer = styled.div`
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  padding-left: 2rem;
+  padding-right: 2rem;
+`;
+
+const ImageContainer = styled.div`
+  @import url('https://fonts.googleapis.com/css?family=Raleway:300,400,700&display=swap');
+  position: absolute;
+  font-family: "Raleway", sans-serif;
+  text-align: center;
+  display: block;
+  max-width: 100%;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+`;
+
+const GridContainer = styled.div`
+  background-image: url('/rectangle26.svg');
+  background-size: cover;
+
+  display: inline-grid;
+  position: relative;
+  grid-template-rows: 50% 50%;
+  grid-template-columns: 40% 30% 30%;
+  grid-template-areas:
+      "sidebar content content"
+      "sidebar content1 content1";
+  text-align: center;
+  border-radius: 40px;
+`;
+
+const SideBar = styled.div`
+  grid-area: sidebar;
+  padding: 5px;
+  margin: 15px;
+`;
+
+const ContentBox = styled.div`
+  display: flex;
+  align-items: center;
+  grid-area: content;
+  margin: 0px;
+`;
+const Content1 = styled.div`
+  opacity: 0.9;
+  background-image: url('/Rectangle 29.svg');
+  padding: 1rem;
+  width: 100%;
+  height: 100%;
+  display: block;
+  justify-content: space-between;
+  row-gap: 0;
+`;
+const Content2 = styled(Content1)`
+  background-image: url('/Rectangle 27.svg');
+  border-top-right-radius:40px;
+`
+const Content3 = styled(Content1)`
+  background-image: url('/Rectangle 30.svg');
+`;
+const Content4 = styled(Content1)`
+  background-image: url('/Rectangle 31.svg');
+  border-bottom-right-radius: 40px;
+`;
+
+const ContentBox1 = styled.div`
+  display: flex;
+  gap: 0.25rem;
+  padding: 0.25rem;
+  align-items: center;
+  grid-area: content1;
+  justify-content: center;
+`;
+const Footer = styled.footer`
+  background: #5282FF;
+  padding: 0.25rem;
+  display: inline-grid;
+  text-align: right;
+  width: 80%;
+  bottom: 0;
+  position: fixed;
+  left: 10%;
+`;
```

Comment:
Footer should generally be an external component placed in the `cse-ui-kit` folder